src/components/NavSession/NavSection.tsx

```diff
@@ -14,35 +14,37 @@ type NavItemProps = {
 
 type Props = {
 	data: Array<NavItemProps>,
+	sx?: object,
```

Comment:
'The `Object` type actually means "any non-nullish value", so it is marginally better than `unknown`.',
        '- If you want a type meaning "any object", you probably want `Record<string, unknown> instead.`',
        '- If you want a type meaning "any value", you probably want `unknown` instead.',